src/actions/posts.js

```diff
@@ -951,6 +967,8 @@ export async function getProfilesAndStatusesForPosts(posts, dispatch, getState)
     const statusesToLoad = new Set();
 
     Object.values(posts).forEach((post) => {
+        // https://github.com/facebook/flow/issues/2221
+        // $FlowFixMe - Object.values currently does not have good flow support
```

Comment:
I ran into this exact same issue when trying to annotate this function on one of our feature branches, even when I had the object as `{[string]: Post}`. It's a tough one

Reply:
Yes, but I think there are workarounds to this problem. I think you can do something like:

```javascript
Object.keys(posts).forEach((key: string) => {
    const userId = posts[key].user_id;
    ...
});
```